CRM/Upgrade/Incremental/sql/4.7.8.mysql.tpl

```diff
@@ -13,3 +13,5 @@ DELETE FROM civicrm_state_province WHERE name = 'Fernando de Noronha';
 -- CRM-17118 extend civicrm_address postal_code to accept full data strings from paypal etc.
 ALTER TABLE civicrm_address CHANGE `postal_code` `postal_code` varchar(64) ;
 
+-- CRM-18699 Fix Wake Island misspelling, was Wake Ialand
+UPDATE civicrm_state_province SET name="Wake Island" WHERE id=4981;
```

Comment:
I would recommend doing
` where name='Wake Ialand' `
instead of relying on state_province.id

Also this should be done in 4.7.9.mysql.tpl which is the next scheduled release.